presto-tests/src/test/java/io/prestosql/tests/TestTpchDistributedStats.java

```diff
@@ -40,8 +41,11 @@ public void setup()
             throws Exception
     {
         DistributedQueryRunner runner = TpchQueryRunnerBuilder.builder()
-                // We are not able to calculate stats for PARTIAL aggregations
-                .amendSession(builder -> builder.setSystemProperty(PREFER_PARTIAL_AGGREGATION, "false"))
+                .amendSession(builder -> builder
+                        // We are not able to calculate stats for PARTIAL aggregations
+                        .setSystemProperty(PREFER_PARTIAL_AGGREGATION, "false")
+                        // Distributed query runner does not collect stats for non-EXPLAIN queries by default
+                        .setSystemProperty(COLLECT_PLAN_STATISTICS_FOR_ALL_QUERIES, "true"))
```

Comment:
Thanks for the comment. I still wonder _why_ we have to set it here, in this class.

Reply:
because we need to for stats tests?

Reply:
> which the class name suggest?



as it is (`// Distributed query runner does not collect stats for non-EXPLAIN queries by default`) it just states what the default is, no point in saying it here

i hoped for something more direct, but let it assume it's self-explanatory